docs/sources/explore/explore-inspector.md

```diff
@@ -1,97 +1,87 @@
+View raw request and response in the Query tab.
 
-You can view raw query results, that is the data returned by the query in a table.
+1. Open the Query inspector and click the **Query tab**.
+1. Click **Refresh**.
 
-In the **Inspector** tab, click the **Data** tab.
+Grafana sends the query to the server and displays the result. You can drill down on specific portions of the query, expand or collapse all of it. Click **Copy to clipboard** to copy the data to use in other applications.
 
-For multiple queries or for queries multiple nodes, there are additional options.
+## Query inspector JSON tab
 
-- **Show data frame:** Select the result set data you want to view.
-- **Series joined by time:** View the raw data from all of your queries at once, one result set per column. You can click a column heading to sort the data.
+View data results as JSON and as data frame JSON models in the **JSON tab**.
 
-### Download raw query results as CSV
+1. Open the Query inspector and click the **JSON tab**.
+1. Choose one of the following options from the **Select source** dropdown menu:
+   - **Panel data -** Displays a JSON object representing the data retrieved from Explore.
+   - **DataFrame JSON (from query) -** Displays the raw data result set without transformations and field configuration applied.
 
-Grafana generates a CSV file in your default browser download location. You can open it in the viewer of your choice.
+## Query inspector Data tab
 
-1. In the **Inspector** tab, get raw query results by following the instructions [described in Inspect raw query results](#inspect-raw-query-results).
-1. Refine query settings until you can see the raw data that you want to export.
-1. Click **Download CSV**.
+View, inspect and download raw query results in the **Data tab**.
 
-In order to download a CSV file specifically formatted for Excel, expand **Data options** and then enable the **Download for Excel** toggle before you click the **Download CSV** option.
+1. Open the Query inspector and click the **Data** tab.
+1. Click **Data options** to to view options under **Show data frame**.
+1. Select a data results set from the dropdown menu.
+1. Select **Series joined by time** from the dropdown to view the raw data from all of your queries at once, one result set per column. You can click any column heading to sort the data.
+1. Toggle **Formatted data** to match the format in the panel.
+1. Toggle **Download for Excel** to download a CSV file specifically formatted for Excel.
+1. To download the results to a CSV file click **Download CSV** in the upper right of the Query inspector pane.
 
 ### Download log results as TXT
 
-Grafana generates a TXT file in your default browser download location. You can open it in the viewer of your choice.
+Based on the type of data source (Loki, for example), Grafana generates a TXT file of log raw data results in your default browser download location. You can open it in the viewer of your choice.
```

Comment:
We can also say that if logs are present in the result, there's an option to download logs as text.

<img width="1709" alt="Screenshot 2024-07-10 at 11 50 11" src="https://github.com/grafana/grafana/assets/745532/14130b4c-1774-4f34-846a-0828dce7f105">

![Uploading Screenshot 2024-07-10 at 11.49.34.png…]()

Reply:
Fixed!